addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1672,6 +1648,23 @@ options.registry.anchorName = options.Class.extend({
         }
         this.$target.trigger('content_changed');
     },
+    /**
+     * Copies anchor to clipbboard.
+     *
+     * @private
+     */
+    _copyAnchorToClipboard: function () {
+        const $input = $("<input>");
+        $("body").append($input);
+        $input.val('#' + this.$target[0].id).select();
+        document.execCommand("copy");
+        $input.remove();
+        this.$el.find('we-button').popover('show');
+        this.popoverHideDP
+            .add(concurrency.delay(1500))
+            .then(() => this.$el.find('we-button').popover('hide'))
+            .catch(() => undefined);
```

Comment:
No need of `catch`

Reply:
If you don't catch, you get "uncaught exception: undefined" errors in the console. Do we want that?